### ISSUE TITLE
Female Elf Fighter

### ISSUE THREAD

**Issue by babaissarkar** [wesnoth/wesnoth#9543]
I think we need a female variant of the elf fighter unit, especially with portraits if possible.

I brought up this on Discord/IRC before, but my logic is:
1. **Equalist Society**
   Elves are much more advanced socially, so they'd prefer to be equalists. The only thing breaking this balance is the Elf Fighter. It isn't explained in lore why only males prefer to take up swordsman ship. For the Shaman/Sorceress line, it's vaguely explained. Also for Shaman we have Lords who can use magic, albeit in a worse way. This also allows us to portray a human-like equalist society compared to the more asymmetric humans.
(And no, Drakes and Saurians don't count, because they are non-humanoid with hardly any dimorphism, so they won't create the same psycological effect. Does any player ever notices that Saurian fighters are *females* while Drakes can be either?
3. **The Elf Ranger**
Problem starts here: there are female Archer/Ranger/Avengers and _they can use swords_. This definitely contradicts the previous case. That means it's not disallowed in Elf society for female elves to learn swordplay. Then why are the females excluded? And if we do want to exclude females, drop them from the Archer line as well. (Which is probably a bad idea, but I'm putting this to point something.)
4. **Lack of art**
That's the biggest blocker for this. Mainline simply won't accept low quality or even medium quality art. And I don't think commissioning a portrait just for this would be likely. We don't allow AI art either, so that option is closed as well.

**Comment by babaissarkar** [comment]
My imperfect portrait attempts:
![fighter-female](https://github.com/user-attachments/assets/5b763387-2882-42f4-8c3f-ff4ace1a2af4)
![fighter-female2](https://github.com/user-attachments/assets/bb2c8560-f5e1-4e3f-80c4-986fbe818119)

(Going to remove the hat from the second one.)

**Comment by CelticMinstrel** [comment]
I'd definitely prefer someone to draw one from scratch rather than your basic modifications (which have the same sorta-issue as Harper's new portrait) but those kinda look okay, at least.

Another reason to have a female fighter is for the Elvish Citizen unit. I think that isn't in core at the moment, but I'd like it to be added.

**Comment by knyghtmare** [comment]
I am curious. You mention "equalist society" and bring up elf fighter, but do not consider the elf scout unit.
If we are trying to brand elves as equalists, I think both Elf Fighter and Elf Scout should have female variants.

Closest sprite art (keeping as close to the current elves as possible) you can get for these is from Vyncyn. So, he could be requested, assuming he still active.

**Comment by babaissarkar** [comment]
> but do not consider the elf scout unit.

Sorry for that, I almost forgot about the Scout line. Would be good to have female variants for those as well. That increases the portrait count however. At worst 1 portrait for Scout would probably be enough, but good to have more.

**Comment by Dalas121** [comment]
I would like to note that elves aren't entirely equalist - they're very gender-segregated when it comes to the practice of magic. The Elvish Lord's description reads:

> lords are typically the only male elves who are allowed to learn any amount of faerie magic, which is traditionally taught solely to female shamans

However, since we already have female archers, I have no issue with female fighters. And such a change would allow us to potentially add the Elvish Civilian to core.

**Comment by babaissarkar** [comment]
> I would like to note that elves aren't entirely equalist - they're very gender-segregated when it comes to the practice of magic. The Elvish Lord's description reads:
> 
> > lords are typically the only male elves who are allowed to learn any amount of faerie magic, which is traditionally taught solely to female shamans
> 
> However, since we already have female archers, I have no issue with female fighters. And such a change would allow us to potentially add the Elvish Civilian to core.

Thanks!
For the lords being the only male elves allowed to use magic, IMO we should give a much better reason that just `they are not allowed`.
For example, IIRC UMC Dragon Trilogy mentions that female elves have a much better chance because they are very distant descendents of Vanadya, the source of all magic. (Please pardon my ignorance in case of a mistake, played it a very long time ago)
Or in Genesis, probably because they are related to Fairies or Fae.

A non wesnoth example would be why there are no male Aes Sedai in The Wheel of Time. (at least at the beginning).

Another thing, the new fighter sprites show the fighters having a shield/buckler, possibly a metal one.
1. there's a fairly big difference between fighting with a shield than without one. 
2. i also have problems with *Elves* out of all people using metal, tho they are using metal swords so why not.

In short, the story/lore should closely track the sprites, otherwise they might soon spiral out of control.
